### DISCUSSION TITLE
Aptos build时出现 inline 的错误

### DISCUSSION THREAD

**Discussion by hotboice** [NonceGeek/Move-Camp#254] [Q&A] [answered]
报错提示源码路径为 move-stdlib/sources/option.move:248:12
```
public inline fun for_each<Element>(o: Option<Element>, f: |Element|)
```
Unexpected 'inline', Expected a module member: 'spec', 'use', 'friend', 'const', 'fun', or 'struct'

**Comment by qiwihui** [comment]
有项目代码吗？方便复现

**Reply by hotboice** [reply]
估计是aptos新更新的一个关键字 ~ 我修改了rev，已经可以pubulish啦 ~

**Comment by 99Kies** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/45918704/215416523-9d86b8ae-7320-458d-a914-6746d84ec036.png)
https://github.com/aptos-labs/aptos-core/commit/415b466f3c0371c502703c2b093dfdeaa73a6996
aptos在3天前添加了inline关键词

遇到这个问题需要更新aptos cli
然后rev写成 mainnet或者devnet
